enterprise/internal/licensing/features.go

```diff
@@ -60,6 +60,15 @@ func checkFeature(info *Info, feature Feature) error {
 	return nil // feature is activated for current license
 }
 
+func MockLicenseCheckErr(expectedError string) {
```

Comment:
```suggestion
func MockCheckFeatureError(expectedError string) {
```

I think this is more consistent name.